package.json

```diff
@@ -23,6 +23,7 @@
   },
   "dependencies": {
     "asynciterator": "^1.1.0",
+    "commander": "^2.20.0",
```

Comment:
This one doesn't seem necessary anymore.

Reply:
true. I anyway prefer minimist, which might be worth considering to keep the commandline a bit under control.

Reply:
Fixed.